docs/src/languages/php/composer-auth.md

```diff
@@ -27,33 +32,36 @@ List that repository in your `composer.json` file.
 }
 ```
 
-## Set a project variable
+## 2. Set up Composer authentication using a variable
 
-Set the Composer authentication by adding a project level variable called `env:COMPOSER_AUTH` as JSON and available only during build time.
+To allow Composer to successfully authenticate when accessing the declared private repository,
+set an `env:COMPOSER_AUTH` variable for your project.
```

Comment:
:nail_care: It might be worth linking this to https://docs.platform.sh/development/variables.html